libs/wingc/src/lsp/completions.rs

```diff
@@ -857,11 +931,24 @@ fn get_completions_from_namespace(
 			}
 		}
 	}
+
+	let is_wing_file = matches!(namespace.module_path, ResolveSource::WingFile);
+
 	namespace
 		.envs
 		.iter()
 		.flat_map(|env| env.symbol_map.iter())
 		.flat_map(|(name, symbol)| format_symbol_kind_as_completion(name, &symbol.kind))
+		// ignore the builtin-functions
+		.filter(|c| {
+			if is_wing_file && c.kind == Some(CompletionItemKind::FUNCTION) {
+				c.label == UtilityFunctions::Assert.to_string()
+					&& c.label == UtilityFunctions::Log.to_string()
+					&& c.label == UtilityFunctions::Assert.to_string()
```

Comment:
Can we use `UtilityFunctions::all()` to keep this code DRY?

Reply:
Filtering to public-only fixes this as well